### DISCUSSION TITLE
language customize

### DISCUSSION THREAD

**Discussion by leminhkien00** [pdfme/pdfme#98] [Ideas]
I think that you should add on the Customize feature  to remake Value in i18n for people using libraby can custommize as their wants easily

**Comment by hand-dot** [comment]
Yes, true.

I created this issue. https://github.com/pdfme/pdfme/issues/107
We'll implement the language customize.

**Comment by hand-dot** [comment]
this is implemented by https://github.com/pdfme/pdfme/pull/323
